lib/beaker/options/command_line_parser.rb

```diff
@@ -194,6 +194,20 @@ def initialize
             @cmd_options[:fail_mode] = mode =~ /stop/ ? 'fast' :  mode
           end
 
+          opts.on '--only-failures',
```

Comment:
This option has actually been superseded by dealing with this logic at the task level in the [ci.rake file](https://github.com/puppetlabs/beaker-puppet/blob/master/tasks/ci.rake) in beaker-puppet. I plan on pulling it out, but wanted to get this review request out.

Reply:
And it's gone...